examples/bval-evaluation-redeployment/runner/pom.xml

```diff
@@ -47,6 +47,20 @@
     <dependency>
       <groupId>org.apache.cxf</groupId>
       <artifactId>cxf-rt-frontend-jaxrs</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>javax.annotation</groupId>
+          <artifactId>javax.annotation-api</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>javax.ws.rs</groupId>
+          <artifactId>javax.ws.rs-api</artifactId>
+        </exclusion>
+        <exclusion>
```

Comment:
I'm getting this error:
```
Tests in error:
  RedeploymentTest: Could not invoke deployment method: public static org.jboss.shrinkwrap.api.Archive RedeploymentTest.webapp1()
```
The logs: 
[lo2.log](https://github.com/apache/tomee/files/2915539/lo2.log)

Reply:
Hi, thanks again for checking.

So I'm wondering why when I run the full tests from the parent (as the buildbot build does), it does not fail but when I run the example project directly from itself own sub-folder like you have done, it does.

I've looked at the example/pom.xml and it appears that some examples are not included as modules.  I see 165 example folders and only 158 modules listed in examples\pom.xml.

Missing examples are:
applet
bval-evaluation-redeployment
java-modules
mp-jsonb-configuration
mp-metrics-metered
mp-opentracing-traced
mp-rest-jwt-principal

Do you know if all of these are excluded for a reason?

Specifically bval-evaluation-redeployment is not being tested by buildbot.

Reply:
interesting finding.

> Missing examples are:
> applet
> bval-evaluation-redeployment
> java-modules
> mp-jsonb-configuration
> mp-metrics-metered
> mp-opentracing-traced
> mp-rest-jwt-principal

I don't know why this is excluded for a reason.

 I'm going to open a new thread about these missing modules on the mailing list because it's outside of the scope of this PR.

Reply:
Okay, thanks!